portal/index.php

```diff
@@ -484,7 +484,7 @@ function callServer(action, value = '', value2 = '') {
                 data: data
             }).done(function (rtn) {
                 if (action === "cleanup") {
-                    window.location.href = "./index.php" // Goto landing page.
+                    window.location.href = "./index.php?site=<?php echo urlencode($_SESSION['site_id']); ?>" // Goto landing page.
```

Comment:
would use the fancy-schmancy js_url() function here:
```
window.location.href = "./index.php?site=" + <?php echo js_url($_SESSION['site_id']); ?>; // Goto landing page.
```

Reply:
I don't mind changing....